commons/commons-model-extended-serialization/src/test/kotlin/uk/ac/ebi/extended/serialization/serializers/ExtSubmissionSerializerTest.kt

```diff
@@ -0,0 +1,143 @@
+package uk.ac.ebi.extended.serialization.serializers
+
+import com.fasterxml.jackson.core.JsonGenerator
+import com.fasterxml.jackson.databind.JsonSerializer
+import com.fasterxml.jackson.databind.Module
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializerProvider
+import com.fasterxml.jackson.databind.module.SimpleModule
+import ebi.ac.uk.dsl.json.JsonObject
+import ebi.ac.uk.dsl.json.jsonArray
+import ebi.ac.uk.dsl.json.jsonObj
+import ebi.ac.uk.extended.model.ExtAttribute
+import ebi.ac.uk.extended.model.ExtProcessingStatus
+import ebi.ac.uk.extended.model.ExtSection
+import ebi.ac.uk.extended.model.ExtStat
+import ebi.ac.uk.extended.model.ExtSubmission
+import ebi.ac.uk.extended.model.ExtSubmissionMethod
+import ebi.ac.uk.extended.model.ExtTag
+import ebi.ac.uk.extended.model.Project
+import io.github.glytching.junit.extension.folder.TemporaryFolderExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import uk.ac.ebi.extended.serialization.constants.ExtSerializationFields
+import uk.ac.ebi.serialization.extensions.serialize
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+@ExtendWith(TemporaryFolderExtension::class)
+class ExtSubmissionSerializerTest {
+    private val testInstance = ObjectMapper().registerModule(createModule())
+
+    @Test
+    fun `serialize basic section when not released`() {
+        val extendedSubmission = testSubmission(false)
```

Comment:
I think adding the name parameter would make the test clearer, i.e. `testSubmission(released = false)`